### ISSUE TITLE
Add Environment Variable To Disable Auto Start

### ISSUE THREAD

**Issue by mkochendorfer** [docksal/docksal#1009]
Auto starting containers when you hit their domain is awesome.  But sometimes browsers with many open tabs can be starting up containers randomly unexpectedly in the background.  It would be great to allow power users to disable this feature if they find it undesirable via a global env var in docksal.env.  Depending on your setup, the containers could start eating all of your system resources, so would be good to give people more tools to manage this.

Basically the inverse of https://github.com/docksal/docksal/issues/685
